packages/search-token-server/app.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-namespace */
+import * as express from 'express';
+import {ensureTokenGenerated} from './middlewares/searchToken';
+import {errorHandler} from './middlewares/errorHandler';
+import {environmentCheck} from './middlewares/environmentCheck';
+
+const app = express();
+
+declare global {
+  namespace Express {
+    interface Request {
+      token: string;
+    }
+  }
+}
```

Comment:
I really don't like to mix the declaration with the code, can you split that into a declaration file?